app/services/employment/RemoveEmploymentService.scala

```diff
@@ -96,18 +93,13 @@ class RemoveEmploymentService @Inject()(deleteOrIgnoreEmploymentConnector: Delet
     nrsService.submit(user.nino, nrsPayload.toNrsPayloadModel, user.mtditid, user.trueUserAgent)
   }
 
-  private def handleConnectorCall(taxYear: Int, employmentId: String, toRemove: String,
-                                  allEmploymentData: AllEmploymentData, isLastEmployment: Boolean, user: User)
+  private def handleConnectorCall(taxYear: Int, employmentId: String, toRemove: String, user: User)
                                  (implicit hc: HeaderCarrier): Future[Either[APIErrorModel, Unit]] = {
 
     deleteOrIgnoreEmploymentConnector.deleteOrIgnoreEmployment(user.nino, taxYear,
       employmentId, toRemove)(hc.withExtraHeaders("mtditid" -> user.mtditid)).flatMap {
```

Comment:
Could this be a map and just return Left or Right?

Reply:
Yes, this was the suggestion that I made during the group review.